### DISCUSSION TITLE
Installation problem

### DISCUSSION THREAD

**Discussion by Richmet706** [nxrighthere/UnrealCLR#15] [Help] [answered]
Hi.Would you help me with an installation issue?
The plugin is installed but cannot be initialized. I have restarted the engine.
gUnrealCLR: Error: UnrealCLR::Module::HostError: The specified runtimeconfig.json [C:/Users/richm/Documents/Unreal Projects/Practice/Plugins/UnrealCLR/Managed/UnrealEngine.Runtime.runtimeconfig.json] does not exist
LogUnrealCLR: Error: UnrealCLR::Module::StartupModule: Unable to initialize the host! Please, try to restart the engine.

**Comment by nxrighthere** [comment]
Tell me please your OS, .NET Core SDK version, the current path from where you are running the installation, and the full path to your Unreal Engine project.

**Comment by Richmet706** [comment, marked as answer]
I got everything working. The managed build did not run successfully due to a path issue.

**Comment by nxrighthere** [comment]
Glad to hear!
